pandas/tests/indexes/test_base.py

```diff
@@ -1615,6 +1615,15 @@ def test_string_index_repr(self):
 
                 self.assertEqual(coerce(idx), expected)
 
+    def test_is_index_empty(self):
+        ''' GH 13207 Bug Fix '''
```

Comment:
this needs to go in ``indexes/tests/common.py`` in ``Base`` so it will generically test all index types

further use ``"""`` rather than ```'''`` as doc-string markers